api_types.ts

```diff
@@ -776,6 +776,29 @@ export interface TemporaryBlobStorage {
   ): Promise<string>;
 }
 
+
+/**
+ * A service for retrieving the sync state in Coda Brain.
+ * @hidden
+ *
+ * TODO(ebo): unhide this
+ */
+export interface SyncStateService {
+  /**
+   * Retrieve the latest version of rows with the given row ids and returns a mapping of row ids to
+   * their latest version, e.g. {"Id-123": "1.0.0"}.
+   *
+   * If the row id is not found, it will not be included in the response.
+   *
+   * If the row version is not defined, it will be set to an empty string.
+   *
+   * @hidden
+   *
+   * TODO(ebo): unhide this
+   */
+  getLatestRowVersions(rowIds: string[]): Promise<{[rowId: string]: string}>;
```

Comment:
We call things items and item ids in most places, but we use "rows" and row ids" here, how come the difference?

Reply:
We felt like it would be better to standardize on "rows". Here's the accompanying change, it's pretty small: https://github.com/coda/packs-sdk/pull/3129

Reply:
We also already used rows:
![Screenshot 2024-12-09 at 5 12 50 PM](https://github.com/user-attachments/assets/3a59b342-bda4-4911-a988-f67964084157)